can-view-parser.js

```diff
@@ -328,6 +335,9 @@ HTMLParser.parseAttrs = function(rest, handler){
 		lookingForValue: false,
 		lookingForEq : false
 	};
+	//maps end characters to start characters
+	var startOppositesMap = {"{": "}", "(":")"};
```

Comment:
can we pull this out into the outer scope so it doesn't have to be re-generated every time the function is called?